src/couch/src/couch_bt_engine_header.erl

```diff
@@ -261,6 +261,7 @@ upgrade_uuid(#db_header{} = Header) ->
     end.
 
 upgrade_epochs(#db_header{} = Header) ->
+    Node = init_node_name(),
```

Comment:
I think we call upgrade_epoch for every header open, parse, update params, etc. This code will call get_arguments which could bottleneck on the init process. Granted, the init process is not doing anything much anyway, but we also don't want risk it, it could affect other parts of the system.